platform/ext/target/nordic_nrf/nrf5340/partition/region_defs.h

```diff
@@ -34,7 +34,7 @@
 
 /* Size of nRF SPU (Nordic IDAU) regions */
 #define SPU_FLASH_REGION_SIZE   (0x00008000)
-#define SPU_SRAM_REGION_SIZE   (0x00002000)
+#define SPU_SRAM_REGION_SIZE    (0x00002000)
```

Comment:
Could you add such definition for nRF9160 on the occasion (it's missing)?

Reply:
done